docs/integrations/sources/instagram.md

```diff
@@ -1,67 +1,59 @@
 # Instagram
 
-This page guides you through the process of setting up the instagram source connector.
-
+This page contains the setup guide and reference information for the Instagram source connector.
 
 ## Prerequisites
```

Comment:
Reword the Prerequisites to be a list instead of instructions to maintain consistency with other connector docs
(example: Create a Meta for Developers account > Meta for Developers account)